src/random.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+function random() {
+  let checkedValue = false;
+  const randomNum = Math.ceil(Math.random() * 10000);
+
+  for (let i = 0; i < randomNum.length; i++) {

```

Comment:
randomNum is a number, not a string. You cannot use .length property or indexing on a number. Convert randomNum to a string before this line.

Reply:
still not fixed